src/main/java/ar/com/escuela/programacion/eventplanner/EventplannerApplication.java

```diff
@@ -47,9 +50,12 @@ private static void executeSelectedOption(String userInput) {
         if (EXIT_VALUE.equalsIgnoreCase(userInput)){
             return;
         } else if ("A".equalsIgnoreCase(userInput)) {
-            printOptionA();
+            OptionA optionA = new OptionA("EventsMaster2023");
+            optionA.printUserNameMessage();
         } else if ("B".equalsIgnoreCase(userInput)) {
-            printOptionB();
+            OptionB optionB = new OptionB();
+            optionB.setNextEventDate("29/12/2569");
+            optionB.printNextEventDateMessage();
         } else if ("C".equalsIgnoreCase(userInput)) {
             printOptionC();
```

Comment:
Acá deberías haber instanciado un nuevo objeto de tipo OptionC y pedirle a ESE objeto que imprima el texto.

```suggestion
            OptionC optionC = new OptionC("Federico Saurio", "Nana Costas", "Nati Chispita (PO)", "Lucas Costas (PO-LLO)");
            optionC.printNextEventGuestListMessage();
```